### DISCUSSION TITLE
SANs

### DISCUSSION THREAD

**Discussion by TonyJHART** [Venafi/venafips#106] [Ideas]
It would be brilliant to be able to use a txt file that has a list of SANs
This my already be . 
To be able to "Add SANs to an 'Existing' cert".  
I honestly have not looked into how to add a SAN ,so the Private Key remains unchanged.

**Comment by gdbarron** [comment]
Hi @TonyJHART.  Unfortunately, no plans to read from a text file directly.  You would need to write a couple lines to read it in and convert to hashtable.

I'm not sure if you can or how you would update and keep the key unchanged, if possible, so hopefully someone else can chime in.
